src/core/link-to-dfn.js

```diff
@@ -132,7 +132,10 @@ export async function run(conf, doc, cb) {
     // The `possibleExternalLinks` above doesn't include references that
     //   match selectors like `a[data-cite="spec"]`
     ...document.querySelectorAll(
-      "a[data-cite]:not([data-cite='']):not([data-cite*='#'])"
+      [
```

Comment:
Recommendation: Array vs backticks vs `+` here

Reply:
If all we are doing here is trying to avoid a long line, then just:
``
...document.querySelectorAll(
  "a[data-cite]:not([data-cite='']):not([data-cite*='#'])," +
  "dfn:not([data-cite='']):not([data-cite*='#'])"
);
```

Then no need for the array.